www/_posts/2021-02-03-cordova-ios-release-6.2.0.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author:
+    name: Niklas Merz
+    url: https://twitter.com/niklasmaerz
+title:  "Cordova iOS 6.2.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.2.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.2.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
```

Comment:
Shouldn't we fill in those section?

Reply:
I put this line at the wrong place.

What are the highlights of this release? Should we mention more. Mostly small stuff as far as I see.

Reply:
I'm fine if we "label" it as some sort of maintenance release - sometimes updates are not that fancy, but nevertheless important.